Library/Homebrew/formula.rb

```diff
@@ -127,6 +127,11 @@ class Formula
   # @private
   attr_accessor :local_bottle_path
 
+  # When performing a build, test, or other loggable action, indicates which
+  # log file location to use.
```

Comment:
Strictly speaking, this is now the log file prefix, not the log file location, right? Maybe it would even make sense to explicitly enumerate all the values this can be set to in this documentation comment.